katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-%global prever .rc1
+%global prever .rc1.1
```

Comment:
as discussed the tag 2.4.0.rc1 was already in the git repo for katello installer, so best option was to go to rc1.1